### DISCUSSION TITLE
Open/Close Photo details pane

### DISCUSSION THREAD

**Discussion by benjithompson** [photoview/photoview#420] [Ideas]
The photo details pane should have the ability to be closed and should be closed by default until explicitly opened using the (i) icon. 

I did a quick search for this and I apologize if I missed this being already discussed or on the roadmap.

**Comment by rickysarraf** [comment]
This is an annoying problem on the mobile view. When you go back from the full screen view of an image, half the mobile screen is grabbed by `details pane`. The `details pane` should be off by default, imho.

**Comment by viktorstrate** [comment]
This will be added with https://github.com/photoview/photoview/pull/382
